pages/asks.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useMemo, ReactElement } from "react";
+import styles from "../styles/pages/asks.module.scss";
+import content from "../server/asks";
+import Ask from "../components/Ask";
+import CreateAskForm from "../components/CreateAskForm";
+import Modal from "../components/Modal";
+
+interface AskContent {
+	content: string,
+	isLiked: boolean,
+	isAnonymous: boolean,
+	userId: string
+}
+
+
+const contentWithTypes: Record<number, AskContent> = content; 
```

Comment:
This is a big big big big and also big no no.
Don't **ever** create variables for type sake only, I really do mean it my friend.

Typescript checks your code at compilation time and is transpiled to javascript.
In the end you'll have 2 variables which are the same and that is
![image](https://github.com/zoop-de-loop/ask.io/assets/98516489/037feb76-35ca-4143-8f6c-4180e0d93e2c)

Reply:
this type should be infered from the content variable itself.

Reply:
noted